packages/react/src/components/Menu/MenuItem.tsx

```diff
@@ -242,18 +243,25 @@ class MenuItem extends AutoControlledComponent<WithAsProp<MenuItemProps>, MenuIt
       menu && active && menuOpen ? (
         <>
           <Ref innerRef={this.menuRef}>
-            {Menu.create(menu, {
-              defaultProps: {
-                accessibility: submenuBehavior,
-                className: MenuItem.slotClassNames.submenu,
-                vertical: true,
-                primary,
-                secondary,
-                styles: styles.menu,
-                submenu: true,
-                indicator,
-              },
-            })}
+            <Popper
+              align={vertical ? 'top' : 'start'}
+              position={vertical ? 'after' : 'below'}
+              targetRef={this.itemRef}
+              modifiers={{ flip: { flipVariationsByContent: true } }}
```

Comment:
This modifier was introduced in `1.15.0`, we should increase required PopperJS version
https://github.com/FezVrasta/popper.js/releases/tag/v1.15.0

Reply:
This is not going to break anything since we still download `1.15.0` (see yarn.lock); will do after OSS approval: #1403 for tracking